src/advanced-search/__tests__/AdvancedSearchContainer.spec.jsx

```diff
@@ -44,10 +44,10 @@ describe('AdvancedSearch component', () => {
     expect(props.dispatchfetchEvidencesIfNeeded).toHaveBeenCalledWith('annotation');
   });
 
-  test('should submit a query', () => {
-    wrapper.find('#submit-query').simulate('click');
-    expect(props.dispatchSubmitQuery).toHaveBeenCalled();
-  });
+  // test('should submit a query', () => {
```

Comment:
You could test it by creating the mock below and checking it was called:
`const historyMock = { push: jest.fn() };`